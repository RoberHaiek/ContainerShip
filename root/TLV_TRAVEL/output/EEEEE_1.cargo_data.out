container9,unload,0,0,3
container10,load,0,0,2