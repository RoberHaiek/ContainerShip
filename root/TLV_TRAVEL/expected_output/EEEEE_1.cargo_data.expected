container9,U,1,1,1
container10,L,1,1,1